hr_timesheet_name_customer/models/hr_timesheet_name_customer.py

```diff
@@ -0,0 +1,14 @@
+from odoo import api, fields, models
+
+
+class NameCustomer(models.Model):
+    _inherit = "account.analytic.line"
+
+    name_customer = fields.Char()
+    """override create method, initialize name_customer"""
+
+    @api.model
+    def create(self, vals_list):
+        if vals_list["name_customer"] is False:
```

Comment:
Just do 
`If not vals.get(“name_customer”):`

Because if name_customer is not present in Val’s vals[“name_customer”] will trigger an exception